e2e-tests/cypress/e2e/dashboard.cy.js

```diff
@@ -137,10 +137,12 @@ describe("Dashboard Test", () => {
    it("should test notification page", () => {
     cy.visit("/dashboard/notifications")
     cy.contains('deleted dataset')
-    cy.get('#notificatoin').check();
+    cy.get('#notificatoin').check({ force: true });
```

Comment:
is it #notificatoin ?

Reply:
Ah, I didn't even notice. Those tests were existing and I just fixed it. Looks like the element itself had a typo. Pushing another commit now.